log/lvl.go

```diff
@@ -0,0 +1,385 @@
+// Package log is an output-library that can print nicely formatted
```

Comment:
Huge parts of lvl.go look indeed very familiar. Using git mv here and modifying the file would have helped a lot seeing the changes. Now it looks like a completely new file.